app/Http/Controllers/AdminDash.php

```diff
@@ -883,15 +900,21 @@ public function updateFileDispOrder(Request $request) {
         $dispString = "";
         $files = File::where('type', $request->typ)->orderBy('disp_order', 'ASC')->get();
         foreach ($files as $f) {
-            $dispString .= "<tr>
-                                <td>$f->name</td>
+            $dispString .= "<tr>";
+            if ($f->row_separator) {
+                $dispString .= "<th class=\"text-center\" colspan=\"3\">$f->name</th>";
+            } else {
+                $dispString .= "<td>$f->name</td>
                                 <td>$f->desc</td>
-                                <td>$f->updated_at</td>
```

Comment:
I know this was already here, but I think it would be better to move this logic into the view. The controller really should only be passing data and the view should be doing all of the displaying

Reply:
I looked at this and I'm not sure that it can be moved to a view. This function is triggered by an AJAX call - it is replying with HTML to insert into an existing view when a staff member changes the display order of items in the file browser. If this were in a view, each action would need to trigger a page reload (instead of a smoother AJAX experience) or I would need to have JS rebuild the view (messier, IMO).

Reply:
Hmm yeah I see what you're saying now... I'm not a huge fan of this because it looks messy but definitely messier if JS reloads the view in this case... so yeah let's leave this for now but I wonder if there's a cleaner way to use a JS package to just update the data and handle the view refreshing for us